app/models/manageiq/providers/ansible_tower/automation_manager/configuration_script/api_create.rb

```diff
@@ -8,7 +8,7 @@ def create_in_provider(manager_id, params)
 
       # Get the record in our database
       # TODO: This needs to be targeted refresh so it doesn't take too long
-      EmsRefresh.queue_refresh(manager, nil, true) if !manager.missing_credentials? && manager.authentication_status_ok?
+      EmsRefresh.refresh(manager) if !manager.missing_credentials? && manager.authentication_status_ok?
```

Comment:
@bdunne - Should this be extracted into another method? - There is a test failure here - I changed it so we don't break the transaction on https://github.com/syncrou/manageiq/blob/01f67760b0bc98060720df6a3d1c44e6d54685ef/app/models/service_template_ansible_playbook.rb#L60

But it seems there may be other callers expecting it to queue the refresh.